autoload/lsp/completion.vim

```diff
@@ -269,13 +268,6 @@ export def CompletionResolveReply(lspserver: dict<any>, cItem: any)
   var infoText: list<string>
   var infoKind: string
 
-  if cItem->has_key('detail') && !cItem.detail->empty()
```

Comment:
Why is this block of code removed?

Reply:
sorry my mistake  
fixed in https://github.com/yegappan/lsp/pull/230/commits/1888f32088a5303c07d009b283f58b6b6a1e130b